File: JS/UberVan.js
const Car = require('./Car')

class UberVan extends Car {
  constructor(license, driver, typeCarAccepted, seatsMaterial) {
    super(license, driver)
    this.typeCarAccepted = typeCarAccepted
    this.seatsMaterial = seatsMaterial
  }
}

module.exports = UberVan

File: Python/UberVan.py
from Car import Car


class UberVan(Car):
    typeCarAccepted = []
    seatsMaterial = []

    def __init__(self, license, driver, typeCarAccepted, seatsMaterial):
        super.__init__(license, driver)
        self.typesCarAccepted = typeCarAccepted
        self.seatsMaterial = seatsMaterial